css/global.css

```diff
@@ -551,9 +551,9 @@ button,
 }
 
 button:hover,
-button:focus,
+button:focus-within,
 .btn-main:hover,
-.btn-main:focus {
+.btn-main:focus-within {
```

Comment:
trying to address buttons staying focused after click. still need a fix for this